site/sfguides/src/vhol_iceberg_snowflake_open_catalog/vhol_iceberg_snowflake_open_catalog.md

```diff
@@ -0,0 +1,575 @@
+               "Condition": {
+                  "StringLike": {
+                        "s3:prefix": [
+                           "*"
+                        ]
+                  }
+               }
+            }
+      ]
+   }
+```
+
+    Before clicking save, replace “<my_bucket>” with the name of the bucket you specified in step 2.e.i (i.e., deconnect-hol-demo-<name>) in both places. Note that the value you replace with this should just be the name of the bucket (“deconnect-hol-demo-<name>”) and not include the protocol (i.e., “s3://”)
+    In the bottom right, click “Next”
+    For “Policy name”, enter “deconnect-hol-demo-policy”
+    Scroll down, and in the bottom right click “Create policy”
+
+<!-- ------------------------ -->
+
+## Create an IAM role to grant privileges on your S3 bucket
+
+- In the navigation pane on the left under “Access management”, select “Roles”
+- Select “Create role”
+- For “Trusted entity type”, select “AWS account”
+- Under “An AWS account”:
+    Select the radio button for “This account (<account_id>)”
+    Check the box that says “Require external ID”
+    In the text box that shows up after you check that box, enter “deconnect_hol_demo”
+    Click “Next”
+- On the “Add permissions” screen, search for and check the box next to the name of the policy created in step 3.f.v (“deconnect-hol-demo-policy”)
+- Click “Next”
+- For “Role name”, enter “deconnect-hol-demo-role”
+- Scroll down and click “Create role”
+- On the next screen, the “Roles” overview page, click on the name of the role you just created (“deconnect-hol-demo-role”)
+- In the “Summary” box, under “ARN”, click the copy icon to copy the role’s ARN to your clipboard. Paste this value in the HOL spreadsheet
+
+<!-- ------------------------ -->
+
+## Creating a Catalog in Snowflake Open Catalog
+
+- Go to your browser tab that you’re logged into Snowflake Open Catalog (or if you’ve closed it, in a new browser tab, go to the account locator URL you copied earlier)
+- On the left pane, click “Catalogs”
+- In the upper right, click the “+ Catalog” button
+- On the create catalog dialog that pops up:
+    For “Name”, enter “external_catalog_snowflake”
+    For “External”, enable the toggle so it’s “on”   ** VERY IMPORTANT**
+    For “Storage provider”, select “S3”
+    For “Default base location”, enter the S3 URI you copied in step 2.j (keep the “s3://” prefix)
+    Leave “Additional locations” blank/empty
+    For “S3 role ARN”, enter the role ARN you copied in step 4.j
+    For “External ID”, enter the external ID you specified in step 4.d.iii (“deconnect_hol_demo”)
+    Click “Create”
+- This will bring you to the main “Catalogs” page. Select the catalog you just created (“external_catalog_snowflake”)
+- Under the “Storage details” section, click the copy icon next to “IAM user arn”. Paste this value in the HOL spreadsheet
+
+<!-- ------------------------ -->
+
+## Grant the Snowflake Open Catalog IAM user permissions to access bucket objects
```

Comment:
@vinodhini-sd Shorten and title case the step label so the left-nav looks cleaner